### DISCUSSION TITLE
How to disable unused {imports, pattern, etc...} diagnostic reporting?

### DISCUSSION THREAD

**Discussion by jigargosar** [elm-tooling/elm-language-server#658] [Q&A]
Many times I like to keep unused code around, since I will be using it soon.

And would like to quickly reach compiler errors in a file, rather than worring about unused.

If there was a way to toggle these diagnostic messages, i.e. only show compiler errors. It would make navigating and fixing broken code much easier.

Any workaround?

**Comment by razzeee** [comment]
You should be able to use a `elm-analyse.json` to configure those.

Just create the file next to your `elm.json` and then vscode should help you filling it.
